pkgs/development/libraries/libetpan/default.nix

```diff
@@ -1,31 +1,29 @@
 { stdenv, fetchFromGitHub
-, autoconf
-, automake
-, libtool
-, openssl
+, autoconf, automake, libtool, openssl, pkg-config
 }:
 
 stdenv.mkDerivation rec {
   pname = "libetpan";
-  version = "1.9.3";
+  version = "1.9.4";
 
   src = fetchFromGitHub {
     owner = "dinhviethoa";
     repo = "libetpan";
     rev = version;
-    sha256 = "19g4qskg71jv7sxfxsdkjmrxk9mk5kf9b6fhw06g6wvm3205n95f";
+    sha256 = "0g7an003simfdn7ihg9yjv7hl2czsmjsndjrp39i7cad8icixscn";
   };
 
-  nativeBuildInputs = [ libtool autoconf automake ];
+  nativeBuildInputs = [ autoconf automake libtool pkg-config ];
 
   buildInputs = [ openssl ];
 
   configureScript = "./autogen.sh";
 
   meta = with stdenv.lib; {
-    description = "An efficient, portable library for different kinds of mail access: IMAP, SMTP, POP, and NNTP";
+    description = "Mail Framework for C Language";
```

Comment:
```suggestion
    description = "Mail Framework for the C Language";
```

Reply:
Grammatically yes, but this is the official description in the project's GitHub repository.

Reply:
It doesn't have to be exactly the same ;).

Reply:
Okay, I have changed it